fairlearn/datasets/_fetch_data.py

```diff
@@ -0,0 +1,91 @@
+import warnings
+from ._common import FairnessWarning
+
+from sklearn.datasets import fetch_openml
+
+
+def fetch_boston(*, version='1', cache=True, data_home=None,
```

Comment:
I played around with this a little because I'm interested in adding more datasets once you've established the pattern. However, the version argument seems to only be relevant when passing "name" as opposed to the `data_id` that's used below.

Reply:
Yep. Discovered that. Should be fixed in next commit.